src/Http/Controllers/CP/Assets/BrowserController.php

```diff
@@ -71,7 +71,9 @@ public function folder(Request $request, $container, $path = '/')
 
         $folder = $container->assetFolder($path);
 
-        $assets = $folder->queryAssets()->paginate(30);
+        $assets = $folder->queryAssets()
+            ->orderBy($request->sort ?? 'basename', $request->order ?? 'asc')
```

Comment:
Can you wrap `orderBy` in an `if` for whether `sort` is in the request?

It'll probably save overhead by not ordering if it doesn't need to.

Reply:
Sure!